src/components/List/Row.js

```diff
@@ -90,79 +104,86 @@ const ListItem = ({
-          >
-            {dateTitle}
-          </BoldText>
-          <SecondaryText>{dateSubTitle}</SecondaryText>
-        </Column>
-
-        <MobileOnlyColumn small={8}>
-          <Column medium={4} large={3}>
-            <PrimaryText>{title}</PrimaryText>
-          </Column>
-          <Column>
-            <SecondaryText>{subTitle}</SecondaryText>
-          </Column>
-        </MobileOnlyColumn>
-
-        <DesktopOnlyColumn medium={8} large={4.5} xLarge={4.9}>
-          <PrimaryText>{title}</PrimaryText>
-        </DesktopOnlyColumn>
-        <SubTitleColumn large={4.5} xLarge={4.9}>
-          <PrimaryText>{subTitle}</PrimaryText>
-        </SubTitleColumn>
-
-        <DesktopOnlyColumn medium={2} large={1.6} xLarge={1.1}>
-          <ListRowButton
-            aria-label={buttonText}
-            role="button"
-            width="102px"
-            variant="standard"
-          >
-            {buttonText}
-          </ListRowButton>
-        </DesktopOnlyColumn>
-
-        <MobileOnlyColumn small={1}>
-          <ClickableButton onClick={onOverflowClick}>
-            <OverflowIcon
-              size={22}
-              color={colors.onyx.light}
-              onClick={onOverflowClick}
-            />
-          </ClickableButton>
-        </MobileOnlyColumn>
-      </ListRow>
-    </LinkWrapper>
+  <RowContainer variant={variant} onExpand={onClick}>
+    <ListRow>
+      <Column small={10.6} medium={9.8} large={10.2} xLarge={10.8}>
+        <LinkWrapper onClick={onClick} href={href}>
+          <Row>
+            <Column small={3.8} medium={2.1} large={1.5} xLarge={1}>
+              <BoldText
+                style={{ textTransform: "uppercase" }}
+                color={coloredDate ? colors.heliotrope.base : null}
+              >
+                {dateTitle}
+              </BoldText>
+              <SecondaryText>{dateSubTitle}</SecondaryText>
+            </Column>
+
+            <MobileOnlyColumn small={7}>
+              <Column medium={4} large={3}>
+                <PrimaryText>{title}</PrimaryText>
+              </Column>
+              <Column>
+                <SecondaryText>{subTitle}</SecondaryText>
+              </Column>
+            </MobileOnlyColumn>
+
+            <DesktopOnlyColumn medium={8} large={4.5} xLarge={4.9}>
+              <PrimaryText>{title}</PrimaryText>
+            </DesktopOnlyColumn>
+            <SubTitleColumn large={4.5} xLarge={4.9}>
+              <PrimaryText>{subTitle}</PrimaryText>
+            </SubTitleColumn>
+          </Row>
+        </LinkWrapper>
+      </Column>
+
+      <DesktopOnlyColumn medium={2} large={1.6} xLarge={1.1}>
+        <ListRowButton
+          aria-label={buttonText}
+          role="button"
+          width="102px"
+          variant="standard"
+        >
+          {buttonText}
+        </ListRowButton>
+      </DesktopOnlyColumn>
+
+      <MobileOnlyColumn small={1.4}>
+        <ClickableButton
```

Comment:
For this instance it's using `padding-top: ${spacing.cozy}` & `padding-top: ${spacing.cozy}` it doesn't really work if we want an expanded tap area. can you make the padding for this button `padding-top: 14px` & `padding-top: 14px`